extra_foam/gui/plot_widgets/image_view_base.py

```diff
@@ -120,14 +123,20 @@ def __init__(self, *,
     def initUI(self):
         layout = QHBoxLayout()
         layout.addWidget(self._plot_widget, 4)
-        layout.addWidget(self._hist_widget, 1)
+
+        if self._histogram:
+            layout.addWidget(self._hist_widget, 1)
+
         layout.setSpacing(0)
         layout.setContentsMargins(0, 0, 0, 0)
         self.setLayout(layout)
 
     def reset(self):
         self.clear()
 
+    def addMenuItem(self, *args, **kwargs):
```

Comment:
I believe it would be better to follow the signature of the parent `addMenuItem`, rather than using `*args, **kwargs`

Reply:
Fixed in 30eb8d96.